daiquiri/__init__.py

```diff
@@ -88,8 +88,27 @@ def setup(level=logging.WARNING, outputs=[output.STDERR], program_name=None,
         root_logger.removeHandler(handler)
 
     # Add configured handlers
-    for o in outputs:
-        o.add_to_logger(root_logger)
+    for out in outputs:
+        # instantiating an output from a dict of options
+        if isinstance(out, dict):
```

Comment:
Not convinced by this dict approach. It seems this is just using
`{'syslog': {'facility': 'user1'}}`
Rather than
`output.Syslog(facility="user1")`

which to be honest is not a win. Maybe just ditch it entirely?